core/src/main/java/io/opentelemetry/android/export/BufferedDelegatingExporter.kt

```diff
@@ -0,0 +1,85 @@
+package io.opentelemetry.android.export
+
+import io.opentelemetry.sdk.common.CompletableResultCode
+import java.util.concurrent.atomic.AtomicBoolean
+
+/**
+ * An in-memory buffer delegating span exporter that buffers span data in memory until a delegate is set.
+ * Once a delegate is set, the buffered span data is exported to the delegate.
+ *
+ * The buffer size is set to 5,000 by default. If the buffer is full, the exporter will drop new span data.
+ */
+internal abstract class BufferedDelegatingExporter<T, D>(private val bufferedSignals: Int = 5_000) {
```

Comment:
```suggestion
internal abstract class BufferedDelegatingExporter<T, D>(private val maxBufferedSpanData: Int = 5_000) {
```
I would rename this to match what's in the subclasses and the docs here.

Reply:
The docs are wrong. This is the abstract class that applies to all signals. ill rename it to `maxBufferedSignals` and fix the docs.